package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Visualisez, triez, et améliorez votre boite mail !",
   "author": "Fabrique des ministère sociaux",
   "license": "Apache-2.0",
-  "main": "index.js",
+  "main": "dist/main/main.js",
```

Comment:
`index.ts` ?

Reply:
Avec la config `main` dans le package.json on doit cibler le compilé car electron ne peux pas interpréter ts as is. C'est utilisé notamment par le point d'entrée des tests e2e.